### DISCUSSION TITLE
How do I contribute a docset?

### DISCUSSION THREAD

**Discussion by ccmywish** [zealdocs/zeal#1493] [Q&A] [answered]
I've noticed that Zeal lacks some docs, but it exists in this repo https://github.com/Kapeli/Dash-User-Contributions/tree/master/docsets

Should I directly make contributions to that repo or somewhere else?

**Comment by trollixx** [comment, marked as answer]
The best option is to actually submit docset to Dash User Contributed repo you linked. Then it can be added to Zeal via https://zealusercontributions.vercel.app/.
